pkgs/development/libraries/libwnck/default.nix

```diff
@@ -21,8 +18,16 @@ stdenv.mkDerivation rec {
 
   configureFlags = [ "--disable-introspection" ]; # not needed anywhere AFAIK
 
+  passthru = {
+    updateScript = gnome3.updateScript {
+      packageName = pname;
+    };
```

Comment:
This is actually an unmaintained version branch of `libwnck` so `updateScript` is pointless here. Will need to carry out https://github.com/NixOS/nixpkgs/issues/40015, but that is a separate issue.

Reply:
Oh oops. I tried to avoid updating old version, but here the old file is called `default.nix`.